functests/2_performance_update/updating_profile.go

```diff
@@ -156,7 +157,7 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 						Size:  hpSize2M,
 					},
 					{
-						Count: 3,
```

Comment:
In the case of multiple numa nodes (Baremetal), the when there is odd hugepage count, i found that it doesn't distribute it evenly, in the sense that it puts 1 hugepage in node0 and 1 in node1, i.e it consumes only 2 ,  it doesn't seem to do 1 in node0 and 2 in node1. So my guess was its because of odd hugepage count. So i increased it to 3.